### DISCUSSION TITLE
Jellyfin can't play .ts containered AAC recorded audio file

### DISCUSSION THREAD

**Discussion by tds101** [jellyfin/jellyfin#9202] [General]
I have an issues where recorded internet radio files don't play properly in Jellyfin. I record the audio stream (http://wbur-sc.streamguys.com/wburlive) using NextPVR, which creates a .ts file, which is encapsulated AAC file). That file plays fine in VLC and NextPVR, but when I add the directory to a library in Jellyfin and try to play on the local computer, a remote computer with Jellyfin client, or Jellyfin client on my Android phone, it starts to play then stutters, then posts "Playback Error. This client isn't compatible with the media and the server isn't sending a compatible media format"

The .m3u file I imported the channel with is this

#EXTM3U
#EXTINF:-1,WBUR-FM
http://wbur-sc.streamguys.com/wburlive

I've tried a few other stations and get the same error.

Also Jellyfin doesn't recognize the .ts files when I add a library as "Content type"  Music, so I have to add the library as Movies, then they show up in Jellyfin.

Anyone thoughts on what could be wrong.

I'm running Jellyfin and nextpvr on a Windows 10 desktop. Latest version of Jellyfin (10.8.9) and NextPVR (6.1.1.221106)

tds

**Comment by tds101** [comment]
Following up on this. I still can't get the .aac recorded IPTV audio file to stream via Jellyfin.

it is saved as a .ts file by NextPVR, and plays fine locally using VLC, but can't get it to play locally or on a remote jellyfin client. It starts to play then give the Playback Error ... client isn't compatible...

Below is the Media Info from jellyfin.



WBUR-BBC Newshour_20230206_10001100

Containermpegts

PathC:\Users\Public\NPVR-recordings\Manual Recordings\WBUR-BBC Newshour_20230206_10001100.ts

Size69 MB

Audio

TitleAAC - Stereo

CodecAAC

ProfileLC

Layoutstereo

Channels2 ch

Bitrate130 kbps

Sample rate48000 Hz

DefaultNo

ForcedNo

ExternalNo

**Comment by pabloocs** [comment]
Hello tds101,

Same issue occurs with aac encapsulated in transport stream for Chromecast devices. The ts files do not play correctly in many places. I recommend you to extract the aac from the ts and have a aac audio manifest instead. That should play in all devices.

**Comment by tds101** [comment]
I was hoping keep things simple as I have all of my recordings automated to record daily and only keep the last 3 days recordings.  As I don't always listen to them I wanted to limit transcoding and play the .aac stream in it's raw format.

I found that NextPVR allows remote streaming of the containerized (.ts) file as well as live streaming of the stream in its raw .aac format, so this works for me for now.  Also I can download, remotely from Jellyfin or NextpVR, the .ts files to my phone and play them using VLC. Although I have to wait for the stream to complete recording to do this.

The NextPVR team mentioned that although Jellyfin should support containerized .aac files and streams, one work around would be to use an "Extra" (not sure what this is) which would transcode to .mp3.  Not sure how to do that though.

 Apparently Jellyfin it doesn't probe the .ts file to determine that it's a .aac audio file.  Jellyfin does play/stream .ts containerized .mpeg2 video file fine.

Would be great if Jellfin supported containerized audio files like it does for video.

tds
